standard/patterns.md

```diff
@@ -129,28 +131,28 @@ designation
     ;
 ```
 
-Given a *pattern input value* *e*, if *designation* is the *identifier* `_`, it denotes a discard ([§9.2.9.1](variables.md#9291-discards)), and the value of *e* is not bound to anything. (Although a declared variable with that name may be in scope at that point, that named variable is not seen in this context.) If *designation* is any other identifier, at runtime the value of *e* is bound to a newly introduced local variable ([§9.2.9](variables.md#929-local-variables)) of that name whose type is the static type of *e*, and the pattern input value is assigned to that local variable.
+Given a pattern input value ([§11.1](patterns.md#111-general)) *e*, if *designation* is the *identifier* `_`, it denotes a discard ([§9.2.9.1](variables.md#9291-discards)), and the value of *e* is not bound to anything. (Although a declared variable with that name may be in scope at that point, that named variable is not seen in this context.) If *designation* is any other identifier, at runtime the value of *e* is bound to a newly introduced local variable ([§9.2.9](variables.md#929-local-variables)) of that name whose type is the static type of *e*, and the pattern input value is assigned to that local variable.
 
 It is an error if the name `var` would bind to a type where a *var_pattern* is used.
 
-## 11.3 Pattern Subsumption
+## 11.3 Pattern subsumption
 
-In a switch statement, it is an error if a case’s pattern is *subsumed* by the preceding set of unguarded cases (XREF).
+In a switch statement, it is an error if a case’s pattern is *subsumed* by the preceding set of unguarded cases ([§13.8.3](statements.md#1383-the-switch-statement)).
 Informally, this means that any input value would have been matched by one of the previous cases.
-Here we define when a set of patterns *subsumes* a given pattern.
+Here is defined when a set of patterns *subsumes* a given pattern.
```

Comment:
I'm not keen on this sentence at the moment. How about "The following rules define when a set of patterns *subsumes* a given pattern:"